### PR TITLE
patch: embellish cantabular server error

### PR DESCRIPTION
**Pull request by saminahbab**
### What

Add cantabular error string type to represent particular errors from the database. 
Ammend component tests to reflect new scenario

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func Get() (*Config, error) {
 				IsSSL: false,
 			},
 		},
-		CensusObservationsFF: false,
+		CensusObservationsFF: true,
```
Comment (j-s-rawat) [inline config/config.go:77]: we will keep this flag defaulted to false in config

File: handler/utils.go [hunk 2]
```diff
@@ -18,3 +18,18 @@ func filterPopulationTypes(coundition []string, toFilter []contract.PopulationTy
 
 	return filtered
 }
+
+type CantabularError string
+
+const (
+	MaxVariableError = "Maximum variables at MSOA and above is 5"
```
Comment (j-s-rawat) [inline handler/utils.go:25]: Where is this text? Local query when run with more than 5 variables returns `Maximum variables in query is 5`

Comment (saminahbab) [reply handler/utils.go:25]: My local gives that error message

**Review by j-s-rawat — APPROVED** [review]
LGTM

**Pull request merged**